lib/db/mysql/patches/patch-018-019.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE codes ADD COLUMN keysJwe VARCHAR(1024),
```

Comment:
Are we confident `1024` will be enough for anybody, or should be make this a `MEDIUMTEXT` or similar column?  Either way, let's ensure we set a length limit on it during request validation so it doesn't get truncated in the db.